src/templates/cromwell/cromwell-resources.template.yaml

```diff
@@ -331,6 +331,16 @@ Resources:
                   - "secretsmanager:CreateSecret"
                   - "secretsmanager:UpdateSecret"
 
+        - PolicyName: !Sub CromwellServer-Notification-Access-${AWS::Region}
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              Effect: Allow
+              Resource: "*"
```

Comment:
Is there any way that the `Resource` could be limited to the actual SNS topic of EventBridge or would that not be known at this time?

Reply:
that will not be known. User should add the topic or bus name in `cromwell.conf` file